R/input-file.R

```diff
@@ -2,30 +2,33 @@
 #'
 #' Create a file upload control that can be used to upload one or more files.
 #'
-#' Whenever a file upload completes, the corresponding input variable is set
-#' to a dataframe. See the `Server value` section.
+#' Whenever a file upload completes, the corresponding input variable is set to
+#' a dataframe. See the `Server value` section.
+#'
+#' Uploaded files are written to a temporary directory. When the user's session
+#' ends, all of the files uploaded in that session will be deleted.
```

Comment:
I think it's worth mentioning that files are stored in a per-user-session temporary directory. Without the qualification it sounds a bit like all files end up in the same temporary directory.

```suggestion
#' Uploaded files are written to a temporary directory unique to each 
#' user and session. When a user's session ends, all of the files uploaded
#' in that session will be deleted.
```

Reply:
Thanks, I agree that's helpful information. There's a detail that needs to be changed, which is that there isn't a separate directory for each session -- each upload has its own directory in R temp dir, and the session keeps track of all such directories and then deletes them when the session ends. I'll do that locally and will push the changes.